Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -863,6 +866,12 @@ extension HomepageViewController: HomepageViewModelDelegate {
     }
 }
 
+extension HomepageViewController: NTPSeedCounterDelegate {
+    func didTapSeedCounter() {
+        SeedCounterNTPExperiment.trackTapOnSeedCounter()
```

Comment:
Similarly here I guess this could be handled inside the cell, though I guess in this case we know in the future we'll want to present the detail screen form here so might be worth to leave it.